### ISSUE TITLE
winrepo.update_git_repos on a masterless windows minion fails with KeyError git.latest

### ISSUE THREAD

**Issue by TheBigBear** [saltstack/salt#20732]
On a masterless windows minion, with file_client set to local and the git repo lines from the unix master added to bottom of windows minion conf file I get:

```
KeyError: 'git.latest'
	Line 1956:   File "salt/modules/win_repo.py", line 114, in update_git_repos
```  

Here is the 'debug' output:

```
C:\salt>salt-call.exe  --local winrepo.update_git_repos -l debug
[DEBUG   ] Reading configuration from c:\salt\conf\minion
[DEBUG   ] Configuration file path: c:\salt\conf\minion
[DEBUG   ] Reading configuration from c:\salt\conf\minion
[DEBUG   ] Mako not available
[ERROR   ] An un-handled exception was caught by salt's global exception handler:
KeyError: 'git.latest'
Traceback (most recent call last):
  File "<string>", line 6, in <module>
  File "__main__.py", line 726, in <module>
  File "__main__.py", line 332, in bootstrap
  File "__main__.py", line 359, in chainload
  File "__main__.py", line 715, in _chainload
  File "__main__.py", line 128, in <module>
  File "__main__salt-call__.py", line 14, in <module>
  File "salt/scripts.py", line 122, in salt_call
  File "salt/cli/__init__.py", line 419, in run
  File "salt/cli/caller.py", line 225, in run
  File "salt/cli/caller.py", line 129, in call
  File "salt/modules/win_repo.py", line 114, in update_git_repos
KeyError: 'git.latest'
Traceback (most recent call last):
  File "<string>", line 6, in <module>
  File "__main__.py", line 726, in <module>
  File "__main__.py", line 332, in bootstrap
  File "__main__.py", line 359, in chainload
  File "__main__.py", line 715, in _chainload
  File "__main__.py", line 128, in <module>
  File "__main__salt-call__.py", line 14, in <module>
  File "salt/scripts.py", line 122, in salt_call
  File "salt/cli/__init__.py", line 419, in run
  File "salt/cli/caller.py", line 225, in run
  File "salt/cli/caller.py", line 129, in call
  File "salt/modules/win_repo.py", line 114, in update_git_repos
KeyError: 'git.latest'
```

Is that not supposed to work? 

Note: I do have a working install of git in the path of that machine.

Here is the nearly 2000 lines of further debugging output in a gist https://gist.github.com/TheBigBear/3fa65236939b68dc7285

**Comment by ssgward** [comment]
Thanks for the report. @TheBigBear what version of the Windows minion are you using? And what version of salt are you using?

**Comment by TheBigBear** [comment]
Windows minion 2014.7.0 (64-bit), and here goes the ```salt-call --versions-report``` output:

```
C:\salt>salt-call --versions-report
           Salt: 2014.7.0
         Python: 2.7.5 (default, May 15 2013, 22:44:16) [MSC v.1500 64 bit (AMD64)]
         Jinja2: 2.7.1
       M2Crypto: 0.21.1
 msgpack-python: 0.4.2
   msgpack-pure: Not Installed
       pycrypto: 2.6
        libnacl: Not Installed
         PyYAML: 3.11
          ioflo: Not Installed
          PyZMQ: 14.1.1
           RAET: Not Installed
            ZMQ: 4.0.4
           Mako: Not Installed
Traceback (most recent call last):
  File "<string>", line 6, in <module>
  File "__main__.py", line 726, in <module>
  File "__main__.py", line 332, in bootstrap
  File "__main__.py", line 359, in chainload
  File "__main__.py", line 715, in _chainload
  File "__main__.py", line 128, in <module>
  File "__main__salt-call__.py", line 14, in <module>
  File "salt/scripts.py", line 122, in salt_call
  File "salt/cli/__init__.py", line 370, in run
  File "salt/utils/parsers.py", line 126, in parse_args
  File "salt/utils/parsers.py", line 194, in print_versions_report
AttributeError: 'module' object has no attribute 'EX_OK'
```

**Comment by ssgward** [comment]
I was also able to reproduce the error.

**Comment by twangboy** [comment]
related to the git.latest issue

**Comment by terminalmage** [comment]
This was fixed in #26148, closing.
